test/unit/selector.js

```diff
@@ -608,6 +610,12 @@ QUnit.test( "attributes - special characters", function( assert ) {
 	assert.deepEqual( Sizzle( "input[data-attr='foo_baz\\']']", null, null, attrbad ),
 		q( "attrbad_injection" ),
 		"string containing quote and right bracket" );
+	assert.deepEqual( Sizzle( "input[value=\\30 \\30\\37 ]", null, null, attrbad ),
```

Comment:
Why does the first escape sequence end with a whitespace and the second occurrence of it does not?

Reply:
Verifying that hexadecimal escape interpretation is correct both with and without a trailing whitespace character.